Dockerfile

```diff
@@ -1,5 +1,5 @@
-# Use the official Node.js 18 image as a base
-FROM node:23-alpine@sha256:498bf3e45a4132b99952f88129ae5429e3568f3836edbfc09e3661515f620837 AS builder
+# Use the official Node.js 22 image as a base
```

Comment:
I'd remove this comment since it's going to be ignored by dependabot hence it may be easily outdated. Furthermore, we use the same image for building and running, so I'm not sure if it still makes sense to have it.

Reply:
should I remove both or just update it to a general form?

Reply:
I'd remove them